ModelCatalogueCorePluginTestApp/src/groovy/org/modelcatalogue/core/elasticsearch/DataTypeDocumentSerializer.groovy

```diff
@@ -24,7 +25,15 @@ class DataTypeDocumentSerializer extends CatalogueElementDocumentSerializer<Data
         }
 
         if (dataType.instanceOf(EnumeratedType) && dataType.enumerations) {
-            safePut(builder, 'enumerated_value', dataType.enumerationsObject.toJsonMap())
+            Map<String, String> enums
+            if (dataType.ext) {
+                enums = new TreeMap<>()
+                enums.putAll(dataType.ext)
+                ((EnumeratedType)dataType).getEnumerations().each {enums.put(it.key, it.value)}
```

Comment:
Couldn't this line be enums.putAll(((EnumeratedType) dataType).getEnumerations())?

putAll appends, it wouldn't overwrite the map.

public Map putAll(Collection entries)
Provides an easy way to append multiple Map.Entry values to a Map.

Parameters:
entries - a Collection of Map.Entry items to be added to the Map.
Returns:
the same map, after the items have been added to it.
Since:
1.6.1